pkg/config/ovn/ovn.go

```diff
@@ -43,7 +41,7 @@ func (o *OVNKubernetesConfig) ValidateOVSBridge(bridge string) error {
 
 func (o *OVNKubernetesConfig) withDefaults() *OVNKubernetesConfig {
 	o.OVSInit.DisableOVSInit = false
-	o.MTU = 1400
+	o.MTU = 1500
```

Comment:
The value is defaulted here and down on line 61. Should we have a const for what the default should be to avoid duplicating the literal value?

Reply:
Added a const for the default mtu size.